### DISCUSSION TITLE
How can I restore everything?

### DISCUSSION THREAD

**Discussion by MikeSesketcharba** [undergroundwires/privacy.sexy#357] [Q&A] [answered]
A year ago or so I manually checked a lot of boxes, and even though I made my Windows 10 installation more private and efficient, I hindered a lot of functionality and accessibility. Now I want to make life easier on myself, un-do everything and try the "standard" or "strict" presets and see how they compare.

I didn't write down which changes I made, so I don't know what I am supposed to restore to return to the initial virgin state of Windows.

Thank you very much!

**Comment by undergroundwires** [comment, marked as answer]
Hi, 

You can choose "Select: **All**" then "Revert: **Selected**". All revert scripts are designed to get you back to the initial OS state. Be careful about stuff in "Remove bloatware" though, they may add a lot of software you do not need. Keep all the system apps.

If there's something that's not working after reversal, it is a bug and feel free to open an issue or send a PR.

**Reply by MikeSesketcharba** [reply]
Thanks! Some things were not restored, and I will open issues accordingly.
